src/devices/nm-device.c

```diff
@@ -6527,6 +6527,46 @@ tc_commit (NMDevice *self)
 			qdisc->parent = nm_tc_qdisc_get_parent (s_qdisc);
 			qdisc->info = 0;
 
+			if (strcmp (qdisc->kind, "fq_codel") == 0) {
+				GVariant *var;
+
+				var = nm_tc_qdisc_get_attribute (s_qdisc, "limit");
+				if (var && g_variant_is_of_type (var, G_VARIANT_TYPE_UINT32))
+					qdisc->fq_codel.limit = g_variant_get_uint32 (var);
```

Comment:
see https://github.com/NetworkManager/NetworkManager/blob/f5e8bbc8e082520af714df290da262ad731ccc4d/src/nm-ip4-config.c#L848 . I like that better.